ui/components/app/dropdowns/network-dropdown.js

```diff
@@ -327,14 +324,15 @@ class NetworkDropdown extends Component {
                     history.push(`${ADVANCED_ROUTE}#show-testnets`);
                   }}
                 >
-                  {t('here')}
+                  {t('showHide')}
                 </a>,
               ])}
               <button
-                title={t('dismiss')}
                 onClick={hideTestNetMessage}
-                className="fas fa-times network-dropdown-content--close"
-              />
+                className="network-dropdown-content--dismiss"
```

Comment:
Nit: this was a pre-existing issue with the `--close` classname, but this name isn't correct according to BEM. The part after the double-dashes is for _modifiers_, like "hover", "pressed", "focused", "disabled", "invalid", etc. Temporary states.

A BEM-compliant name would be `network-dropdown-content__dismiss

Reply:
Well, technicallty `network-dropdown__dismiss` would be better. The "content" part of the network dropdown should be an _element_, because it's not a standalone entity. The entire network dropdown seems like the better "block" in this component.

But that would require changing all of the classnames in this component, which seems even father outside of scope than my first suggestion. Would not recommend for this PR.

Reply:
Very good suggestions thanks, I can update this in a separate PR